imapclient/status.go

```diff
@@ -9,6 +9,10 @@ import (
 )
 
 func statusItems(options *imap.StatusOptions) []string {
+	if options == nil {
+		options = &imap.StatusOptions{}
+	}
```

Comment:
Is this a nil pointer deref you're running into?

If so, can you submit it as a separate PR?

Reply:
OK, I got a panic here

Reply:
I've pushed this change as e767b0db12f2.